google/cloud/forseti/scanner/audit/iam_rules_engine.py

```diff
@@ -152,7 +152,14 @@ def find_violations(
         violations = self.rule_book.find_violations(
             resource, policy, policy_bindings)
 
-        return set(violations)
+        # dedupe violations
```

Comment:
As mentioned in the PR comment, would be great to make this de-dupe feature to be an option, and there is a condition here to check that option before de-duping.  Are you familiar with how to make this optional?  There are examples in the [forseti_conf_server.yaml.in](https://github.com/forseti-security/forseti-security/blob/master/configs/server/forseti_conf_server.yaml.in) file, where the values are [populated by terraform like here](https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/server_config/variables.tf).  If you need more exact instructions, please let us know.